src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
You should consider adding a fallback font in case 'Roboto' or 'Arial' don't load correctly. This could be a generic font like 'serif' or 'sans-serif'.